curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.md

```diff
@@ -22,7 +22,7 @@ Complete the function using the rules below to modify the object passed to the f
 -   Your function must always return the entire `records` object.
 -   If `value` is an empty string, delete the given `prop` property from the album.
 -   If `prop` isn't `tracks` and `value` isn't an empty string, assign the `value` to that album's `prop`.
```

Comment:
```suggestion
-   If `value` is an empty string, delete the given `prop` property from the album.
-   If `prop` isn't the string `tracks` and `value` isn't an empty string, assign the `value` to that album's `prop`.
```

Reply:
That feels awkward. Its more literally accurate, but its also a mouthful

Reply:
Agreed, it's a little awkward perhaps, but isn't it a source of confusion?

Reply:
I haven't noticed it as a particularly big issue on the forum. I've seen more confusion about where/how to check if the `track` property exists (though that could be confirmation bias?)